src/lib.rs

```diff
@@ -524,7 +550,7 @@ pub enum DynFormat {
 #[serde(tag = "tag", content = "data")]
 pub enum Format {
     /// Reference to a top-level item
-    ItemVar(usize, Vec<Expr>),
+    ItemVar(usize, Vec<Expr>), // FIXME - do the exprs here need type(+) info?
```

Comment:
Presumably the author should not need to specify the types of expr here, it would make more sense to specify the type of formats when declaring them if necessary.